yugabyte/src/yugabyte/single_key_acid.clj

```diff
@@ -1,4 +1,7 @@
 (ns yugabyte.single-key-acid
+  "Given a single table of hash column primary key and one value column with 5 rows,
```

Comment:
Is it really 5 rows? Can you double-check the test operations history?

Reply:
You're right, we're only using two rows. Good catch, thanks!

Reply:
Correction - we're using variable number of rows, discussed and updated comment accordingly